src/components/JiraHolder.js

```diff
@@ -119,6 +124,15 @@ const JiraHolder = () => {
 		});
 	};
 
+	const handleDisable = () => {
+		const isEmptyResult = document.querySelector(".alert-secondary");
+		if (isEmptyResult && isEmptyResult.innerHTML){
+			setisDisable(false)
+		}else if (isEmptyResult && isEmptyResult.innerHTML === ""){
+			setisDisable(true)
+		}
```

Comment:
Please use better logic here rather than reading the contents of the resultant DIV. Try something like iterating through the `state.jiras` array and if you see at least one `jira.summary` field defined, then mark the button as enabled.

Reply:
done @anushibin007